core/src/toga/widgets/imageview.py

```diff
@@ -143,11 +146,12 @@ def image(self, image):
         self._impl.set_image(self._image)
         self.refresh()
 
-    def as_image(self, format: type[ImageT] = toga.Image) -> ImageT:
+    def as_image(self, format: type[ImageT] = toga.Image) -> ImageT:  # type: ignore[assignment]
         """Return the image in the specified format.
 
         :param format: Format to provide. Defaults to :class:`~toga.images.Image`; also
             supports :any:`PIL.Image.Image` if Pillow is installed.
         :returns: The image in the specified format.
         """
-        return self.image.as_format(format)
+        # TODO:PR: what's the use-case for initializing with image=None? cause this won't work then...
+        return self.image.as_format(format)  # type: ignore[union-attr]
```

Comment:
Are there established use cases for initializing an `ImageView` with `image=None`?

I'm trying to figure out what to do when `ImageView.as_image()` is called and `image` is `None`...or how to avoid that situation altogether?

Reply:
At least as currently implemented, it's a view that can, but is not *currently* displaying an image. 

In that world, if image is None, I'd argue as_image() also returns None; but I can see the headache that causes for types. 

I guess there's there might be an argument to be made that image *shouldn't* be None... or, at least, if it's allowed in the constructor, Toga provides a placeholder image (maybe even with that placeholder image being configurable)... although I'm a little twitchy about changing/removing features that might otherwise be useful because it's hard to providing type hints for them.